script/Deploy.s.sol

```diff
@@ -44,4 +48,26 @@ contract EvenNumberDeploy is Script {
 
         vm.stopBroadcast();
     }
+
+    function run(string memory deployConfigPath) external {
+        // read and log the chainID
+        uint256 chainId = block.chainid;
+        console2.log("You are deploying on ChainID %d", chainId);
+
+        uint256 deployerKey = uint256(vm.envBytes32("ETH_WALLET_PRIVATE_KEY"));
+
+        vm.startBroadcast(deployerKey);
+
+        // READ JSON CONFIG DATA
+        string memory config_data = vm.readFile(deployConfigPath);
+        address verifier_address = stdJson.readAddress(config_data, ".risc_zero_verifier_address");
+
+        IRiscZeroVerifier verifier = IRiscZeroVerifier(verifier_address);
+        console2.log("Using RiscZeroVerifierRouter contract deployed at", address(verifier));
```

Comment:
This log should reference `IRiscZeroVerifier` since this could actually be implemented by a number of different contracts, depending on the address they use. (Make sure to update the logs copied into the deployment guide)
```suggestion
        console2.log("Using IRiscZeroVerifier contract deployed at", address(verifier));
```